README.md

```diff
@@ -25,6 +25,18 @@ $ cd duo_python
 $ pip install --requirement requirements-dev.txt
 ```
 
+## CRITICAL: Add Duo to logins for `/admin` as well
```

Comment:
Let's change this header to
Enable MFA for admin login

Reply:
Done